pkg/operators/oci-handler/oci.go

```diff
@@ -279,6 +279,8 @@ func (o *OciHandlerInstance) init(gadgetCtx operators.GadgetContext) error {
 
 		subject := strings.Split(annInfo[0], ".")
 		switch len(subject) {
+		case 0:
```

Comment:
0 will never be the case, as the remainder (even if empty) will be returned as first item. The error should only happen `if len(annInfo[])` is 0